### PR TITLE
Switch to new csproj format

### PR DESCRIPTION
**Pull request by aholovko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ Key features
 * Uses the new Airbrake JSON
   (v3)<sup>[[link](https://airbrake.io/docs/#create-notice-v3)]</sup>
 * SSL support (all communication with Airbrake is encrypted by default)
-* Support for .NET 3.5 and above (including the latest .NET Core platforms)
+* Support for .NET 4.5.2 and above (including the latest .NET Core platforms)
+  <sup>[[net35](#net-35-support)]</sup>
```
Comment (kyrylo) [inline README.md]: Link still says `net-35`.

File: README.md [hunk 2]
```diff
@@ -356,26 +353,11 @@ var response = await airbrake.NotifyAsync(ex);
...
-};
-
-airbrake.NotifyAsync(ex);
-```
-
 #### `AddFilter`
 
 A notice can be customized or ignored before it is sent to Airbrake via
 `AddFilter`. A lambda expression that is passed to the `AddFilter` method
-accepts a `Notice` that can be processed by your code. The `Notice` object is
```
Comment (kyrylo) [inline README.md]: Not sure if this is required. It says that it accepts an instance :)

Comment (aholovko) [inline README.md]: Not sure that I got what you meant. Could you please elaborate on this?

Comment (kyrylo) [inline README.md]: The diff says you removed an "a" from "accepts a `Notice`". I think it is OK to have it there.

Comment (aholovko) [inline README.md]: Duh!! Now I see. Ok, I will fix that. )

File: README.md (hunk 1 above)
Comment (aholovko) [inline README.md]: That link is to clarify what is going on with support for .NET 3.5 and 4.5. It resolves to the section ".NET 3.5 Support".

Comment (kyrylo) [inline README.md]: The title is about .NET 4.5, but the link says `#net-35-support`, which is a bit unexpected.

**Pull request merged**